app/controllers/users_controller.rb

```diff
@@ -69,18 +72,6 @@ def update
     end
   end
 
-  def update_language_settings
```

Comment:
we don't really need to handle this separately, now that in the IA it's in the same page as the rest of the Customization section